### PR TITLE
Add DismissRecommendation example

### PR DESCRIPTION
**Pull request by AnashOommen**

### CODE REVIEW

File: examples/Services/Recommendations/DismissRecommendation.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example applies a given recommendation. To retrieve " +
```
Comment (CoryLiseno) [inline examples/Services/Recommendations/DismissRecommendation.cs]: "applies" -> "dismisses"

File: examples/Services/Recommendations/DismissRecommendation.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                    "recommendations for text ads, run GetTextAdRecommendations.cs.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The customer ID for which the call is made.</param>
+        /// <param name="recommendationId">ID of the recommendation to apply.</param>
```
Comment (CoryLiseno) [inline examples/Services/Recommendations/DismissRecommendation.cs]: "to apply" -> "to dismiss"

File: examples/Services/Recommendations/DismissRecommendation.cs (hunk 1 above)
Comment (AnashOommen) [reply examples/Services/Recommendations/DismissRecommendation.cs]: Fixed

File: examples/Services/Recommendations/DismissRecommendation.cs (hunk 2 above)
Comment (AnashOommen) [reply examples/Services/Recommendations/DismissRecommendation.cs]: Fixed.

File: examples/Services/Recommendations/DismissRecommendation.cs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            try
+            {
+                DismissRecommendationResponse response = service.DismissRecommendation(
+                    customerId.ToString(), partialFailure, new DismissRecommendationOperation[] {
+                        operation
+                    });
+                Console.WriteLine($"Dismissed {0} recommendation(s):", response.Results.Count);
+                foreach (DismissRecommendationResult result in response.Results)
+                {
+                    Console.WriteLine($"- {result.ResourceName}");
```
Comment (jradcliff) [inline examples/Services/Recommendations/DismissRecommendation.cs]: Is this how you're printing lists in other examples? Would be good to be consistent.

File: examples/Services/Recommendations/DismissRecommendation.cs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+            bool partialFailure = false;
+
+            try
+            {
+                DismissRecommendationResponse response = service.DismissRecommendation(
+                    customerId.ToString(), partialFailure, new DismissRecommendationOperation[] {
+                        operation
+                    });
+                Console.WriteLine($"Dismissed {0} recommendation(s):", response.Results.Count);
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/Services/Recommendations/DismissRecommendation.cs]: Done.

File: examples/Services/Recommendations/DismissRecommendation.cs [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            DismissRecommendation codeExample = new DismissRecommendation();
+            Console.WriteLine(codeExample.Description);
+
+            //The customer ID for which the call is made.
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/Services/Recommendations/DismissRecommendation.cs]: Done

File: examples/Services/Recommendations/DismissRecommendation.cs (hunk 3 above)
Comment (AnashOommen) [reply examples/Services/Recommendations/DismissRecommendation.cs]: Done

**Pull request merged**